t/t3405-rebase-malformed.sh

```diff
@@ -69,4 +69,11 @@ test_expect_success 'rebase commit with diff in message' '
 	test_cmp G G0
 '
 
+test_expect_success 'rebase exec modifies rebase-todo' '
+        git rebase HEAD -x "echo x \"echo foo >H\" >> \"$(
```

Comment:
And actually, you can leave the test in here if you replace the `git-rebase-todo` file with a single word: `invalid`. Then the rebase should fail. But of course, that might not catch a problem where it fails for the wrong reasons.